packages/modelserver-client/src/model-server-client-v2-integration.spec.ts

```diff
@@ -234,7 +240,7 @@ describe('Integration tests for ModelServerClientV2', () => {
             const notification = await patchNotification;
             const patch = notification.patch;
 
-            expect(notification.modelUri).to.be.equal(modeluri);
+            expect(notification.modeluri.toString()).to.be.equal(modeluri.toString());
```

Comment:
maybe we should use this for comparison: http://medialize.github.io/URI.js/docs.html#equals

Reply:
Great idea, thanks for catching that, will update in upcoming commit.

Reply:
Will revert those again, as we loose information of the uri comparison and would only get an boolean assertion output which is way less informative than the real string comparison (see also https://github.com/eclipse-emfcloud/emfcloud-modelserver-theia/pull/121#discussion_r1019568709).